util/omni_base.hpp

```diff
@@ -0,0 +1,168 @@
+#pragma once
+
+#include "base.hpp"
+
+#include "trajectory.hpp"
+
+namespace hebi {
+namespace experimental {
+namespace mobile {
+
+class OmniBase : public MobileBase {
+public:
+  // Parameters for creating a base
+  struct Params: public MobileBase::Params {
+    Params() : wheel_radius(0.0762), base_radius(0.220), sample_density(0.1) {}
+    double wheel_radius; // m
+    double base_radius; // m (center of omni to origin of base) 
+    double sample_density; // time between sampled waypoints on cartesian trajectory
+  };
```

Comment:
I like the basic idea of the inherited parameter structures here.

I want to try to think of something clean for initialization -- you can always set each value after creating a default instance, or add a constructor whose first argument is an instance of the base class.  I feel like both of those work, but are sort of clunky...I'm not sure.